api.md

```diff
@@ -887,12 +888,51 @@ SELECT detach_tablespaces('conditions');
 ```
 
 ---
+## distributed_exec() [](distributed_exec)
+
+This command is used on an access node to execute a SQL command across
+the data nodes of a distributed database. This command can be used to,
+e.g., create cluster-wide objects across all instances of a
```

Comment:
For stylistic reasons, I avoid "e.g." and "i.e." in text and write "for example" and "that is" instead. Easier on the eye.